javascript-cont/vscode-and-style.md

```diff
@@ -71,139 +71,260 @@ Hello Naya Spence!
 
-![How Coding Standards Proliferate](../assets/learning-another-language__vscode-and-style__coding-stanards.png)
+## Style Guides
 
-*Source: [XKCD](https://xkcd.com/927/)*
+![How Coding Standards Proliferate](../assets/learning-another-language__vscode-and-style__coding-stanards.png)  
+_Source: [XKCD](https://xkcd.com/927/)_
 
 Many software teams adhere to a set of rules which serve as guidelines for how to write and organize code. Linters can be an integral part of a team's style conventions.
 
-### Why use a style guide?
+### Why Use a Style Guide?
 
-Style guides are a common way for a group of people to create consistently formatted documents. Like MLA or the Chicago Manual of Style, a code style guide helps us find the information we are looking for as quickly as possible.
+Style guides are a common way for a group of people to create consistently formatted documents. Similar to how the MLA Guide or the Chicago Manual of Style helps authors clearly present their ideas in writing, a code style guide helps us express our intentions in code in a way that others on our team will be able to understand more easily.
 
-In a large team, with programmers from a lot of different language backgrounds, it's easy to fall into whatever habits your first programming language encouraged. A Python whiz might default to using indentation alone to show where a code block begins and ends, where a C programmer at heart might feel more comfortable surrounding code blocks in curly braces. With small differences like these, one slight difference between two programmers may go unnoticed, but as teams get bigger and more and more styles mix, code becomes increasingly ugly and unreadable. 
+As programmers, it's easy to fall into whatever habits our first programming language or learning materials encouraged. But small differences among programmers on concerns even as minor as spacing around operators, where to place curly braces, or how to name variables can make code more difficult to read. Each time we encounter a difference in style, we have to stop and think about it. Was this done intentionally? Is there a reason for it? Following a style guide helps us avoid these unnecessary context switches.
 
-Simply put, style guides help us keep code formatted consistently between developers, making for fewer context switches, and hopefully fewer silly mistakes.
+For a small team with only a few programmers, slight differences may go unnoticed. But as teams get bigger and more and more styles mix, code becomes increasingly ugly and unreadable. In the most extreme cases it can even lead to conflicts among team members, with time wasted arguing over the "right" way to do things, and needless churn as individuals try to "fix" each other's code, which itself can lead to the introduction of new bugs.
```

Comment:
Could add the word inconsistent here: "But as teams get bigger and more and more styles mix, code becomes increasingly ugly, inconsistent, and unreadable."